ConsentViewController/Classes/WebView.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  WebView.swift
+//  ConsentViewController
+//
+//  Created by Vilas on 8/7/19.
+//
+
+import UIKit
+import WebKit
+import JavaScriptCore
+
+public protocol ConsentWebViewHandler {
+    func willShowMessage()
+    func didGetConsentData(euconsent: String, consentUUID: String)
+}
+
+@objcMembers open class WebView: UIView, UIWebViewDelegate , WKNavigationDelegate, WKScriptMessageHandler {
```

Comment:
What about giving the `WebView` a more descriptive name? Something like `ConsentWebView`.

Reply:
Yeah we can, good suggestion.